src/CypressStringifyExtension.ts

```diff
@@ -89,6 +89,14 @@ export class CypressStringifyExtension extends StringifyExtension {
       );
     }
 
+    if (step.assertedEvents) {
+      step.assertedEvents.map((event) => {
```

Comment:
Do you need to iterate with a .map here? if I'm reading it correctly we're just looping over the assertEvents and taking action if we find the type we want. We don't need the new array that the map would create in memory. Maybe a .forEach or a For of loop is a leaner way to go?

Reply:
Good point, Paul. I switched it out using `forEach`.